webui/src/components/ClipboardButton.js

```diff
@@ -127,4 +128,43 @@ const ClipboardButton = ({ text, variant, onSuccess, onError, icon = <ClippyIcon
     );
 };
 
-export default ClipboardButton;
+
+const RunFilterButton = ({ variant, to, icon = <FilterIcon/>,  tooltip}) => {
```

Comment:
Move to RunFilterButton or/and rename and move to FilterButton

Reply:
Done.